build.psm1

```diff
@@ -689,7 +689,7 @@ function Restore-PSPester
         [ValidateNotNullOrEmpty()]
         [string] $Destination = ([IO.Path]::Combine((Split-Path (Get-PSOptions -DefaultToNew).Output), "Modules"))
     )
-    Save-Module -Name Pester -Path $Destination -Repository PSGallery -RequiredVersion "4.8.0"
+    Save-Module -Name Pester -Path $Destination -Repository PSGallery -RequiredVersion '4.10.1'
```

Comment:
Can we use MaximumVersion here too?

Reply:
Yes, we could. We are only expecting small patches of v4 anyway so it's quite risk free.

Reply:
```suggestion
    Save-Module -Name Pester -Path $Destination -Repository PSGallery -MaximumVersion 4.99
```